Libraries/Components/View/View.js

```diff
@@ -227,35 +227,38 @@ const View = React.createClass({
     onLayout: PropTypes.func,
 
     /**
-     * In the absence of `auto` property, `none` is much like `CSS`'s `none`
-     * value. `box-none` is as if you had applied the `CSS` class:
+     * Controls whether the View can be the target of touch events.
      *
+     *   - 'auto': The View can be the target of touch events.
+     *   - 'none': The View is never the target of touch events.
+     *   - 'box-none': The View is never the target of touch events but it's
+     *     subviews can be. It behaves like if the following classes
```

Comment:
Oh yeah, typo :) there's another one a few lines below since I copy pasted that part.

Will it work if I fix it now since you already shiped it?